lib/menulist.rb

```diff
@@ -0,0 +1,11 @@
+module MenuList
+  MENULIST = {
+    :burger => {:Hamburger => 4.50, :Cheeseburger => 4.60, :Chips => 1.00 },
+    :wine => {:red => 3.00, :white => 3.00, :blue => 10.00},
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)